components/amilia/sources/common/base.mjs

```diff
@@ -0,0 +1,51 @@
+import amilia from "../../amilia.app.mjs";
+
+export default {
+  type: "source",
+  props: {
+    amilia,
+    db: "$.service.db",
+    http: {
+      type: "$.interface.http",
+    },
+  },
+  hooks: {
```

Comment:
Check if is possible to implement `deploy` hook here.

Reply:
For the 4 sources, only 1 really uses `deploy`.
`list-activites` can't for permission reasons
`updated-*` doesn't really make sense IMO, I've seen a client saying that too